test/task_running_resources_test.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2012-2015 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import time
+from contextlib import contextmanager
+
+from helpers import RunOnceTask
+
+import luigi
+import luigi.scheduler
+import luigi.worker
+import luigi.rpc
+import server_test
+
+
+luigi.notifications.DEBUG = True
```

Comment:
Is this copied from anywhere? Is it necessary? You just want to delete this line

Reply:
done